Creating the project Code of Conduct (#2)
PeruibeTEC/Server: .github/CODE_OF_CONDUCT.md

# **Code of Conduct**

This code of conduct dictates the expectations of the PeruibeTec team, for the participants of the PeruibeTec community, as well as dictates the steps to report any unacceptable behavior. The purpose of this code of conduct is to make it clear what we expect from the participants and leaders of this project. Anyone who violates this code can and will be banned from projects.

<br>

### **Policy anti-harassment**

- Offensive comments involving race, sexual orientation, religion, gender or appearance will not be tolerated;
- Use this space exclusively to discuss the project and its technologies;
- Videos, photos or gifs of adult or gore content will not be tolerated;
- Inappropriate terms, offensive or disrespectful behavior or bullying will not be tolerated;

> - Jokes of a sexist, sexist, misogynistic, discriminatory nature, or against any group will not be tolerated.

Reviewer: ```suggestion
- Jokes of a sexist, misogynistic, discriminatory nature, or against any group will not be tolerated.
```

<br>

We hope that these policies are followed both in spirit and in letter

---

<br>

### **Reporting Issues**

> If you witness or experience any behavior mentioned above in the [Policy anti-harassment](#policy-anti-harassment), please contact our team via email: PeruibeTec@gmail.com. All submitted reports will be analyzed. In your report include:

Reviewer: ```suggestion
If you witness or experience any behavior mentioned above in the [Policy anti-harassment](#policy-anti-harassment), please contact our team via email: PeruibeTec@gmail.com .All submitted reports will be analyzed. In your report include:
```

> - Contact information
> - Names of any individuals who are involved. If there are any witnesses, include them as well. His account of what he witnessed. If there are still records, include a link;
> - If possible, a screenshot of what happened
> - Any additional information that may be useful.

Reviewer: ```suggestion
- Contact information;
- Names of any individuals who are involved;
- If there are any witnesses, include them as well.His account of what he witnessed. If there are still records, include a link;
- If possible, a screenshot of what happened;
- Any additional information that may be useful.
```

After completing and submitting this report, a person from the PeruibeTec team will contact you. If the report was from any member of the PeruibeTec team, she will not have access to the case of the report. We will respect requests for confidentiality for the purpose of protecting the victim from abuse.

> Anyone requesting to interrupt unacceptable behavior must obey immediately, the order representative can take any action he deems necessary, including permanently banning the project without notice.

Reviewer: ```suggestion
Anyone requesting to interrupt unacceptable behavior must obey immediately, the order representative can take any action he deems necessary to prevent recurrence. These actions can, but are not restricted or imply, from a conversation in search of retraction to the final ban of the project without prior notice.
```
